ci/gpu/build.sh

```diff
@@ -48,14 +49,19 @@ nvidia-smi
 
 gpuci_logger "Activate conda env"
 . /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
 
-conda install -c conda-forge conda-build -y
+gpuci_logger "Install dependencies"
+gpuci_conda_retry install -y -c rapidsai-nightly \
+    sysroot_linux-64=2.17 \
+    "cudatoolkit=${CUDA_VER}.*" \
+    "python=${PYTHON_VER}.*" \
```

Comment:
Is there any reason why `python` is included here? our CI images should have the correct versions of `python` installed by default.

Reply:
Thanks @ajschmidt8! 

Not a good reason.
I think it can be removed.

Reply:
Do we include `cudatoolkit` as well? If so, maybe we can drop that too?

Curious if we can drop `sysroot_linux-64` (was trying that as a workaround in my older PR). Maybe we don't need that either

Reply:
That's a good question. I will check if `cudatoolkit` is already installed in our images. cudf [seems to include it](https://github.com/rapidsai/cudf/blob/branch-21.06/ci/gpu/build.sh#L77-L83). I will verify regardless. Not sure about `sysroot_linux-64`. I'm not familiar with that package.